Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -94,6 +94,40 @@ void commsWithTelemetry::execute(pathManager* pathMgr)
 {
     GetTelemetryCommands(&_incomingData);
 
+    // incoming angles are radians
+    static float gimbalMaxPitch = 3.14159;
+    static float gimbalMaxYaw = 3.14159/2;
+    // pitch 0 = straight ahead
+    // yaw pi/2 = straight ahead.
```

Comment:
Nice comments!

Can gimbalTargetYaw be negative? If it can, then your calculation on line 115 is valid. Else, you will need to do the following check:

```cpp
if (gimbalTargetyaw < pi/ =2 ) { 
  float gimbalPercentYaw = 50 - 100 * gimbalTargetYaw / gimbalMaxYaw; // Change + and - signs as I am guessing what they should be lol
} else {
 float gimbalPercentYaw = 50 + 100 * gimbalTargetYaw / gimbalMaxYaw;
}
```